support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -109,10 +125,13 @@ export function SupporterPlusCheckout({
 			<Box cssOverrides={shorterBoxMargin}>
 				<BoxContents>
 					{showPriceCards ? (
-						<ContributionsPriceCards />
+						<ContributionsPriceCards
+							paymentFrequency={contributionType}
+							inThreeTierVariant={inThreeTierVariantB}
```

Comment:
Is this correct? We're assigning `inThreeTierVariantB` to a prop labelled `inThreeTierVariant`, the prop name implies this should be `true` if they were in variant A too?

Reply:
Yes, its correctly used but your right, very confusing to load `inThreeTierVariantB` into prop `inThreeTierVariant`.  Have decided this `inThreeTierVariant` prop-drilling really isn't necessary and adds needless lines of code. Have amended to use it directly within `priceCards.tsx`.